starfyre/parser.py

```diff
@@ -277,18 +311,16 @@ def handle_data(self, data):
                 js="",
                 signal=component_signal,
                 uuid=uuid,
+                original_name="TEXT_NODE",
             )
         )
-        
-        parent_node.children.append(wrapper_div_component) 
-
-        print(
-            "parent node",            
-            parent_node.tag,
-            parent_node.children,
-            "for the text node ",
-            data,
-        )
+
+        #We need to check if the wrapped component should go to the custom list aka self.current_children or to parent_node children list
```

Comment:
can you please explain a bit more here?

Reply:
> can you please explain a bit more here?

@sansyrox 
If the `text node` is children of a custom tag aka `<parent></parent>`,
that text node should go to `self.current_children list` that will be the replacement for the `slot`.
If we don't check that, the text node will be added straight to the `parent_node.children` and consequently placed in the wrong order on the final HTML.

Reply:
@SuelenKarbivnychyy 👋 

Can you please articulate this explanation as a code comment? and include it there?